libc/exit/exit.c

```diff
@@ -113,8 +114,22 @@ static void test_sigchldHandler(int signum)
 	test_common.test_handlerFlag = TEST_EXIT_DUMMY_VAL;
 }
 
+/* SIGPIPE signal handler */
+static void test_sigPipeHandler(int signum)
+{
+	struct sigaction sa;
+
+	sa.sa_handler = SIG_DFL;  // Set the handler back to default action
+	sigemptyset(&sa.sa_mask);
+	sa.sa_flags = 0;
+	TEST_ASSERT_EQUAL_INT(0, sigaction(SIGPIPE, &sa, NULL));
```

Comment:
no, You can't jump from the signal handler, asserts are not possible in this context

The same applies to all other handlers.

Reply:
Yea, I missed it.